pkg/local_object_storage/metabase/put.go

```diff
@@ -156,7 +157,7 @@ func handleNonRegularObject(tx *bbolt.Tx, currEpoch uint64, obj objectSDK.Object
 				}
 
 				garbageObjectsBKT := tx.Bucket(garbageObjectsBucketName)
-				err = garbageObjectsBKT.Put(target[:], zeroValue)
+				err = garbageObjectsBKT.Put(addressKey(oid.NewAddress(cID, target), make([]byte, addressKeySize)), zeroValue)
```

Comment:
So, it's not just resync?

Reply:
current state: lock object cannot be removed by gc because of incorrect gc bucket internals -> expired lock object was not removed from blobstor -> it was resynced from time to time -> _correct_ TS cannot resync because of stuck LOCK